README.md

```diff
@@ -1,5 +1,55 @@
 # mobile-android-cri-orchestrator
 
+## About
+
+The Credential Issuer (CRI) Orchestrator coordinates identity proofing capability for the [GOV.UK One Login app](https://github.com/govuk-one-login/mobile-android-one-login-app).
+It builds on the [ID Check SDK](https://github.com/govuk-one-login/mobile-id-check-android-sdk) used in the [GOV.UK ID Check app](https://github.com/govuk-one-login/mobile-id-check-android) with additional functionality including REST API based web-to-app handoff and document selection.
+
+Currently the only CRI supported within the SDK is the [Document Checking CRI (DCMAW)](https://github.com/govuk-one-login/mobile-id-check-async), but this component could be extended in the future to allow for additional app-based CRI‍s.
```

Comment:
The app is publicly branded as ID Check, but the mobile CRI is referred to as "Document Checking" internally since that's what differentiates it from other CRIs across the programme (such as Address, Fraud) which are also for checking identity as a whole.

I am also not a fan of the DCMAW (_"Document Checking Mobile App & Web"_) acronym, but it's fairly commonly used still to refer to the Mobile Pod CRI. I can drop it if you don't think it's needed here.

Reply:
Yeah it's a good point, @sandymay-dd I'd love your take on this too.

Reply:
What's the term we'd rather go with? Every non-mobile person on the programme calls us DCMAW I've found 🙃

Reply:
We're issuing the same credential, but it's a different mechanism for getting there. I'd lean towards keeping with what we had, otherwise we'll have two different names for the same service?

Reply:
Makes sense, in which case, should we reference that in the async readme?

Reply:
yep - I was just doing that now :)

Reply:
Ignore me - if we're going to update it we might want to give it a bit of thought. Raising a tech debt ticket for the ongoing sprint.

Reply:
Awesome 🙂 if we have a ticket for that then I will resolve this thread.

Reply:
Ticket: https://govukverify.atlassian.net/browse/DCMAW-10782